addons/web/static/tests/views/fields/properties_field_tests.js

```diff
@@ -1480,4 +1480,48 @@ QUnit.module("Fields", (hooks) => {
             assert.ok(target.querySelector(".o_test_properties_not_empty"));
         }
     );
+
+    QUnit.test(
+        "properties: confirm the delete of an already deleted property does not throw an error",
+        async function (assert) {
+            async function mockRPC(route, { method, model, kwargs }) {
+                if (["check_access_rights", "check_access_rule"].includes(method)) {
+                    return true;
+                }
+            }
+            await makeView({
+                type: "form",
+                resModel: "partner",
+                resId: 3,
+                serverData,
+                arch: `
+                <form>
+                    <sheet>
+                        <group>
+                            <field name="company_id"/>
+                            <field name="display_name"/>
+                            <field name="properties" widget="properties"/>
+                            <div class="o_test_properties_not_empty" attrs="{'invisible': [('properties', '=', [])]}">
+                                Properties not empty
+                            </div>
```

Comment:
Would no copy/paste that part as it's specific to the other test you copied it from.